lib/web_ui/lib/src/engine/canvaskit/surface_factory.dart

```diff
@@ -46,6 +46,9 @@ class SurfaceFactory {
   /// all painting commands.
   final Surface baseSurface = Surface();
 
+  /// A surface used spcifically for `Picture.toImage` calls.
```

Comment:
It may also be useful to the reader to know why we keep it separate from other surfaces (my understanding is it's because we want to be able to perform `Picture.toImage` even when we've used all surfaces in the pool).